pali/COPYING

```diff
@@ -0,0 +1,19 @@
+voc.txt and output.txt are derived from the Digital Pāḷi Dictionary which is licensed under a Creative Commons License: https://digitalpalidictionary.github.io/
```

Comment:
More precisely:

> [Creative Commons Attribution-NonCommercial 4.0 International License](http://creativecommons.org/licenses/by-nc/4.0/).

The "non-commercial" restriction seems potentially problematic.